csharp-refactoring/RefactoringCouplers/RefactoringCouplers/FeatureEnvy/Correct/Ticket.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace RefactoringCouplers.FeatureEnvy.Correct;
+public class Ticket
+{
+    public Ticket(bool available, int price)
+    {
+        Available = available;
+        Price = price;
+    }
+
+    private bool Available { get; set; }
+    private int Price { get; init; }
+
+    public bool IsAvailable()
+    {
+        return Available;
+    }
+
+    private bool CanAfford(decimal points)
```

Comment:
Since the method is private and both `point` in the `BuyTicket` method and `Price` are of type `int`, we should probably just make this an int here as well. No need for the additional precision when we are not using it.
I think an argument could be made that `Price` could be changed to a decimal, since we are representing a currency.

Reply:
Right, my bad! I wanted to keep it simple and go with `int`s everywhere. I corrected it in both Ticket classes, as well as in the article. Thanks :)